x-pack/plugins/observability/public/components/custom_threshold/components/rule_condition_chart/rule_condition_chart.tsx

```diff
@@ -162,6 +176,60 @@ export function PreviewChart({
     setThresholdReferenceLine(refLayers);
   }, [threshold, comparator, euiTheme.colors.danger, metrics]);
 
+  // Build alert annotation
+  useEffect(() => {
+    if (!annotation) return;
+
+    const annotations: EventAnnotationConfig[] = [];
+
+    if (annotation.endTimestamp) {
+      annotations.push({
+        label: `Alert`,
+        type: 'manual',
+        key: {
+          type: 'range',
+          timestamp: annotation.timestamp,
+          endTimestamp: annotation.endTimestamp,
+        },
+        color: chroma(transparentize('#F04E981A', 0.2)).hex().toUpperCase(),
+        id: uuidv4(),
+      } as EventAnnotationConfig);
```

Comment:
Why do we need to cast here?
Can we define a variable to apply proper typing, like: (same for other instances)
```
const thresholdAnnotation: EventAnnotationConfig = {
  label: 'Alert',
  type: 'manual',
  key: {
    type: 'range',
    timestamp: annotation.timestamp,
    endTimestamp: annotation.endTimestamp,
  },
  color: chroma(transparentize('#F04E981A', 0.2)).hex().toUpperCase(),
  id: uuidv4(),
} 
```